beginner_source/audio_preprocessing_tutorial.py

```diff
@@ -52,6 +52,8 @@
 import requests
 import matplotlib
 import matplotlib.pyplot as plt
+import numpy as np
```

Comment:
Is this used?

Reply:
this is needed for comparison to librosa, which takes an np array as input

Reply:
But I do not see any use of `np.XX`.

Reply:
you're right, importing np is not necessary